packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -21,6 +21,8 @@ @interface FLTImagePickerPlugin () <UINavigationControllerDelegate,
 
 @property(copy, nonatomic) FlutterResult result;
 
+@property(nonatomic) bool single;
```

Comment:
Let's make it an integer, where 1 indicates single. 
So 
`@property(assign, nonatomic) integer maxImagesAllowed;`

The first iteration of picking multiple images don't necessary have to support the maxLimit parameter, but it would be nice to have in the future.

Reply:
Done.